### ISSUE TITLE
Timeperiod 24x7 don't work

### ISSUE THREAD

**Issue by Sismic** [ddurieux/glpi_monitoring#205]
i have 24x7 for all my components and in fact command are send with calendar working day.

**Comment by camlafit** [comment]
Hi

I redo a fresh plugin install, and see a pb with timeperiods export webeservices. Have you try a call to 
*monitoring.shinkenTimeperiods* ? Without any more information, would be an empty array()
I you try before with a *monitoring.shinkenServices* call,  array would be populate.

I presume problem with webservice method/call, all timesperiods are set in PHP session. In some case session looks empty.

In my case I've re-enable my timeperiods.cfg file  to get a full timeperiods.

**Comment by ddurieux** [comment]
Have you shinken modyle "import-glpi" up to date?

**Comment by camlafit** [comment]
Hi

In my case done with *shinken update import-glpi*. From import-glpi 0 timeperiods are imported, with call rest on glpi, I can obtain these with this couple query (monitoring.shinkenServices before monitoring.shinkenTimeperiods). Alone array timepreriods is empty.

**Comment by hahahafr** [comment]
Have you solved this ?
Did you try changing the actual timeperiod hours in glpi ?
It's in Accueil > Configuration > Intitulés > Calendriers or /glpi/front/calendar.php directly.

**Comment by Jm0u** [comment]
i've the same problem with fresh install.
import-glpi up to date to and timeperiod.cfg is empty.
i test the import with file get-config.php
i've you got solution?

**Comment by hahahafr** [comment]
Try to modify the actual time periods hours in glpi, the link is in the
previous post. If it doesn't work you have a bug then.

**Comment by Jm0u** [comment]
I test this solution but not work.
so i used file template.cfg directly from shinken

**Comment by ddurieux** [comment]
Yes, it's because you need load timeperiods in the end (so after get host, services...)
